### ISSUE TITLE
About Dataset directory structure

### ISSUE THREAD

**Issue by Carry-w** [jwgdmkj/LMLT#6]
Hello, the author, your work is quite excellent!
When I want to run the test set through the pre training model provided by you, an error is reported about the storage location of the test set. If you can provide the directory structure of the test set.Then I will be very grateful
Or can you provide a download path with the correct directory structure.
Thank you very much.

**Comment by Carry-w** [comment]
![图片](https://github.com/user-attachments/assets/5527b7d6-7864-4f29-84c3-fad534b916af)
and I want to know how can I get the GPU Mem and the Avg Time (as the table 2 shows) when I run the test datasets through the pretraining model provided by you.
 Thank you very much!!

**Comment by jwgdmkj** [comment]
Thank you for your interest in our project. The directory for the test dataset is as follows. 

testsets ---- B100 ---- HR
                |              |-- LR_bicubic  -- X2(3,4)
                |-  Set5 ---- HR  
                |              |-- LR_bicubic   -- X2(3,4)
                |-  Set14 ---- HR  
                |              |-- LR_bicubic   -- X2(3,4)
...

Additionally, GPU memory usage and inference time can be verified using the following code. 
inference/inference_lmlt.py
As stated in the paper, the reported values were recorded when enlarging a 320x180 or 640x360 image with an x4 or x2 scale. Please note that using a different dataset may yield different results.

**Comment by Carry-w** [comment]
Thank you for your answer.
But I found that the test data set I downloaded from the link you provided is not the same as the actual test data set structure, or is missing. Taking urban100 as an example, the downloaded file only includes 100 images of HR and does not include the required LRx2, LRx3 and LRx4.
So I want to know how I can get the pictures of LRx2, LRx3 and LRx4. Or can you provide correct download links for all test data sets. I would appreciate it if you could.
![图片](https://github.com/user-attachments/assets/8876122c-35cd-43f7-af8a-3d3cea7998c1)

**Comment by Carry-w** [comment]
When I use LMLT_Base_x2 for testing tasks
The results are shown in Figure 1. Here, the psnr and ssim obtained by Set5 and Set14 are inconsistent with the data in the paper, while the other three data sets are consistent. So I guess the reason here is whether the following pictures of testsets/Set5 (14)/HR/are incorrect. So can you provide the source of Set5/Set14 HR pictures here. Thank you very much.
![图片](https://github.com/user-attachments/assets/656e856f-049f-45dd-bfac-d3d2cf82dbe6)
![图片](https://github.com/user-attachments/assets/2ee99dc0-fd97-4bda-9a4e-e8f7cb7ed300)
